server/src/utility/mailer.ts

```diff
@@ -0,0 +1,39 @@
+import nodemailer from "nodemailer";
+import ejs from "ejs";
+
+const sib_pass = process.env.SIB_PASS;
+
+const sendOtpEmail = async (email: string, otp: number) => {
+  try {
+    // Create a transporter
+    const transporter = nodemailer.createTransport({
+      host: "smtp-relay.sendinblue.com",
+      port: 587,
+      auth: {
+        user: "pride.education.website@gmail.com",
+        pass: sib_pass
+      }
+    });
+
+    const otpHtml = await ejs.renderFile(__dirname + "/../views/otp.ejs", {
+      otp: otp
+    });
+    // send mail with defined transport object
+    const info = await transporter.sendMail({
+      from: "pride.education.website@gmail.com", // sender address
+      to: email, // list of receivers
+      subject: "OTP verification", // Subject line
+      html: otpHtml // html body
+    });
+
+    console.log(`Message sent: ${info.messageId}`);
+    return `Message sent: ${info.messageId}`;
```

Comment:
I don't think we need the return statement here. If this is not required, we can maybe remove it.